Makefile

```diff
@@ -0,0 +1,29 @@
+deps:
+	go install github.com/terraform-docs/terraform-docs@v0.16.0
+	go install github.com/hashicorp/terraform-config-inspect@latest
+	curl -L "`curl -s https://api.github.com/repos/terraform-linters/tflint/releases/latest | grep -o -E "https://.+?_linux_amd64.zip"`" -o tflint.zip && \
+		unzip tflint.zip && \
+		rm tflint.zip && \
```

Comment:
thaaanks! always forget that when installing requirements automatization >>>> docs.

Reply:
There's no step in the pre-commit-config, right?